python_modules/libraries/dagster-sdf/tox.ini

```diff
@@ -6,10 +6,11 @@ download = True
 passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE*
 install_command = uv pip install {opts} {packages}
 deps =
+  -e ../../dagster-pipes
   -e ../../dagster[test]
   -e .[test]
 allowlist_externals =
   /bin/bash
   uv
 commands =
-  pytest -c ../../../pyproject.toml -vv {posargs}
+  pytest -c ../../../pyproject.toml -vv {posargs} --pdb
```

Comment:
Is `--pdb` necessary?

Reply:
oh my `--pdb` removal commit wasn't pushed, fixed it